packages/amplify_api/example/ios/unit_tests/RestApiUnitTests.swift

```diff
@@ -27,18 +27,21 @@ class RestApiUnitTests: XCTestCase {
     
     func test_get_returns_success() throws {
         class MockApiBridge: ApiBridge {
-            override func get(request: RESTRequest, listener: ((AmplifyOperation<RESTOperationRequest, Data, APIError>.OperationResult) -> Void)?) -> RESTOperation?{
+            static let body = "{\"success\":\"get call succeed!\",\"url\":\"/items\"}"
+            
+            override func get(request: RESTRequest, listener: RESTOperation.ResultListener?) -> RESTOperation {
```

Comment:
Your above unit tests for swift look really good and well organized!  Do you think these original tests are still necessary or could we just use the new ones that you've created?

Reply:
Thanks! And yeah, they're testing different parts of the code. My tests are only testing the new URLSession behavior.